### PR TITLE
Added About AutoZone, Investor Information and Affiliate Program Page. And resolved conflicts.

### PR DESCRIPTION
**Pull request by A737962**
Please review this PR.

### CODE REVIEW

File: autozone-poc/src/components/AboutUs/MockData/Mock.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+        },
```
Comment (gokulwebdeveloper) [inline autozone-poc/src/components/AboutUs/MockData/Mock.json]: remove all external site files. copy paste in our website

File: autozone-poc/src/components/AboutUs/MockData/Mock.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
...emes\/site\/nir_pid1182\/client\/images\/AZO_12345.jpg\"/></div><\/div><div class=\"row\"><h3>AutoZone, Inc.</h3><p> AutoZone is the nation's leading retailer and a leading distributor of automotive replacement parts and accessories with more than 6,000 stores in US, Puerto Rico, Mexico, and Brazil. Each store carries an extensive line for cars, sport utility vehicles, vans and light trucks, including new and remanufactured hard parts, maintenance items and accessories. Since opening its first store in Forrest City, Ark. on July 4, 1979, the company has joined the New York Stock Exchange (NYSE: AZO) and earned a spot in the Fortune 500.</p></div><div class=\"row\"><h3>Recent News</h3><table class=\"table table-striped\"><thead><tr><th scope=\"col\">Date</th><th scope=\"col\">Title and Summary</th></tr></thead><tbody><tr><td>05\/26\/20</td><td>AutoZone 3rd Quarter Same Store Sales Decrease 1.0%; EPS Decreases 10.0% to $14.39; Announces Actions in Response to COVID-19</td></tr><tr><td>04\/28\/20<\/td><td>AutoZone to Release Third Quarter Earnings May 26, 2020</td></tr><tr><td>04\/07\/20<\/td><td>AutoZone Announces Actions In Response To COVID-19</td></tr></tbody></table></div></div>"
```
Comment (gokulwebdeveloper) [inline autozone-poc/src/components/AboutUs/MockData/Mock.json]: Remove slash and use that in our json. Also remove unwanted CSS class

File: autozone-poc/src/utlis/navigation/menu/Menu.js [hunk 3]
```diff
@@ -85,6 +86,22 @@ const Menu = () => {
             <li>
               <NavLink to="/contact">Contact</NavLink>
             </li>
+            <li className="cn-dropdown-item has-down pr12">
+              <a href="#">About Us</a>
+              <ul className="dropdown aboutUs-dropdown">
+                <li>
+                  <NavLink to="/aboutAutozone">About AutoZone</NavLink>
```
Comment (gokulwebdeveloper) [inline autozone-poc/src/utlis/navigation/menu/Menu.js:175]: Use extact to avoid URL confusion.

**Review by gokulwebdeveloper — CHANGES_REQUESTED** [review]

**Comment by A737962** [comment]
Please review. And all the commits are different in nature. Please merge it as it is.

**Pull request closed without merging**